tests/client/test_http.py

```diff
@@ -724,10 +724,10 @@ def test_username(self):
 class TestCrateJsonEncoder(TestCase):
     def test_naive_datetime(self):
         data = dt.datetime.fromisoformat("2023-06-26T09:24:00.123")
-        result = json.dumps(data, cls=CrateJsonEncoder)
-        self.assertEqual(result, "1687771440123")
+        result = cratedb_json_encoder(data)
+        self.assertEqual(result, 1687771440123)
```

Comment:
That's apparently the only kind of anomaly that can be observed after also switching JSON loading per `orjson.loads`.

```diff
-"1687771440123"
+1687771440123
```

Reply:
It can probably be fixed easily by adjusting the `cratedb_json_encoder` slightly?

Reply:
Excellent. I was dearly hoping for such an answer, that it is actually correct behaviour now.